### PR TITLE
AP-77: Changed characters escaping in Liquibase SQL files.

### PR DESCRIPTION
**Pull request by icrc-toliveira**
Issue: https://bahmni.atlassian.net/browse/AP-77

A MYSQL error was found when using openMRS 2.4.0.
The cause seems to be related with the functions inside the files:
* patientPastAppointments.sql
* patientPastAppointments_v2.sql
* patientUpcomingAppointments.sql
* patientUpcomingAppointments_v2.sql

After a closer look, root of the error seems to be the way some characters are escaped.

Solution:
The solution seems too be using different forms of "" and '' instead of relying on \ to do the character escaping

### CODE REVIEW

**Comment by icrc-toliveira** [comment]
"A MYSQL error was found when using openMRS 2.4.0."
This problem seems to result into the following error at runtime:

image

**Comment by dkayiwa** [comment]
@icrc-toliveira i see that you proposed a solution in the pull request description. Did you try it out?

**Comment by icrc-toliveira** [comment]
@dkayiwa I did update the PR
Instead of duplicating the files, I used the old ones and placed the tag `runOnChange="true"` in the changeSet definition
This way, even if the file was already executed by the liquidbase, it still will run.

**Comment by dkayiwa** [comment]
@icrc-toliveira do you have any changes that require to run again for servers where this changeset was run before?

**Comment by icrc-toliveira** [comment]
@dkayiwa 
As a response to "do you have any changes that require to run again for servers where this changeset was run before?"
I don´t think so, since the problem happens during the insertion (If query is already present there shouldn´t be a problem). 
However, if the new version has a different checksum it will throw an error, that why I used "runOnChange='true'".

**Comment by dkayiwa** [comment]
Does the checksum also include the contents of the referenced file?

**Comment by icrc-toliveira** [comment]
@dkayiwa 
Not  sure if I understand the question
However from what I understand, the liquidbase checksum is generated based on the content of the sql file.

**Comment by dkayiwa** [comment]
@icrc-toliveira are you aware of these merge conflicts?

**Comment by icrc-toliveira** [comment]
@dkayiwa The conflicts should now be solved

**Comment by dkayiwa** [comment]
@icrc-toliveira looks good. If i had merge privileges for this repository, i would have done the needful. 😊

File: api/src/main/resources/patientPastAppointments.sql [hunk 1]
```diff
@@ -1,10 +1,9 @@
 INSERT INTO global_property (property, property_value, description, uuid)
- VALUES ('bahmni.sqlGet.pastAppointments',
-"SELECT
+VALUES ('bahmni.sqlGet.pastAppointments','SELECT
      app_service.name                                                                                AS `DASHBOARD_APPOINTMENTS_SERVICE_KEY`,
      app_service_type.name                                                                           AS `DASHBOARD_APPOINTMENTS_SERVICE_TYPE_KEY`,
-     DATE_FORMAT(start_date_time, \"%d/%m/%Y\")                                                        AS `DASHBOARD_APPOINTMENTS_DATE_KEY`,
-     CONCAT(DATE_FORMAT(start_date_time, \"%l:%i %p\"), \" - \", DATE_FORMAT(end_date_time, \"%l:%i %p\")) AS `DASHBOARD_APPOINTMENTS_SLOT_KEY`,
```
Comment (binduak) [inline api/src/main/resources/patientPastAppointments.sql:7]: @icrc-toliveira @mks-d what is the mysql version that you are facing error with ?

Comment (icrc-toliveira) [reply api/src/main/resources/patientPastAppointments.sql:7]: Hi @binduak 
I´m facing this issue with "Mysql  Ver 14.14 Distrib 5.7.31"

**Review by binduak — APPROVED** [review]

File: api/src/main/resources/patientPastAppointments.sql (hunk 1 above)
Comment (binduak) [reply api/src/main/resources/patientPastAppointments.sql:7]: Tested migration changes with mysql 5.6 Distrib, and the changes are working fine. Approved the PR.

**Pull request merged**